### ISSUE TITLE
html in carrels is referring to non-existant files

### ISSUE THREAD

**Issue by dbrower** [ericleasemorgan/reader#189]
This is true for the htm/proper-nouns.htm file in any carrel. To be specific we will choose a carrel at random.

Visiting https://distantreader.org/carrels/subject-socialists-gutenberg/htm/proper-nouns.htm will request two images that don't exist:
* https://distantreader.org/carrels/subject-socialists-gutenberg/images/sort_both.png
* https://distantreader.org/carrels/subject-socialists-gutenberg/images/sort_desc.png

**Comment by ericleasemorgan** [comment]
> This is true for the htm/proper-nouns.htm file in any carrel. To be specific we will choose a carrel at random.
> 
> Visiting https://distantreader.org/carrels/subject-socialists-gutenberg/htm/proper-nouns.htm will request two images that don't exist:
> 
>   • https://distantreader.org/carrels/subject-socialists-gutenberg/images/sort_both.png
>   • https://distantreader.org/carrels/subject-socialists-gutenberg/images/sort_desc.png\


Yes, this is a small but real issue. All of the pages similar to proper-nouns.htm are implemented using the DataTables Javascript library, and the library points to one or two tiny image files looking like triangles. These triangles denote the ability to sort things in ascending or descending order. The links in the implementation do not point to the images' canonical location, which is somewhere in /js  --Eric
